assets/js/hubsetup.js

```diff
@@ -878,7 +872,8 @@ class KubernetesConfigBuilder extends ConfigBuilder {
       spec: {
         selector: {app: 'keycloak'},
         ports: [
-          {protocol: 'TCP', port: 8080}
+          {protocol: 'TCP', port: 8080},
+          {protocol: 'TCP', port: 9000}
```

Comment:
Does this need to be exposed? Can't healthchecks run on localhost:9000 inside the pod?

Reply:
As there is no such thing as an ingress route exposing the port to the outside of the cluster, it is only available for internal communication within the cluster, which is also required for Hub to communicate with this deployment. Without this service it would not be exposed to the cluster network.

In https://gitlab.skymatic.de/skymatic/infrastructure/hub-testing/-/blob/main/deployments/hubtesting_common.yml?ref_type=heads#L307-L315 we go one step further and expose it to the public, protected behind basic auth, to monitor liveness/readiness, but here port 9000 is not exposed to the public.